src/app/modules/main/chat_section/chat_content/messages/view.nim

```diff
@@ -35,7 +35,7 @@ QtObject:
     result.chatColor = ""
     result.chatIcon = ""
     result.chatType = ChatType.Unknown.int
-    result.loading = false
+    result.loading = true
```

Comment:
This doesn't seem to have any effect yet, even so, I would still leave it in true mode by default.